source/docs/software/actuators/overview.rst

```diff
@@ -2,20 +2,22 @@
 
 Actuator Overview
 ==================
-This section discusses the control of motors and pneumatics through speed controllers, solenoids and pneumatics, and their interface with C++ and Java WPILib.
+This section discusses the control of motors and pneumatics through motor controllers, solenoids and pneumatics, and their interface with C++ and Java WPILib.
 
-Speed controllers
+Motor Controllers
 -----------------
-A speed controller is responsible on your robot for making motors move. For brushed DC motors such as CIMs or 775s, the speed controller regulates the voltage that the motor receives, much like a light bulb. For brushless speed controllers such as the Spark MAX, the controller regulates the power delivered to each "phase" of the motor.
+A motor controller is responsible on your robot for making motors move. For brushed DC motors such as CIMs or 775s, the motor controller regulates the voltage that the motor receives, much like a light bulb. For brushless motor controllers such as the Spark MAX, the controller regulates the power delivered to each "phase" of the motor.
+
+.. note:: Another name for a Motor Controller is a Speed Controller.
```

Comment:
```suggestion
.. note:: Another name for a motor controller is a speed controller.
```

Reply:
Reanmed to match the manual specific motor controllers are capitalized generic are lower case